src/main/java/gov/cabinetoffice/api/security/ExceptionHandlerFilter.java

```diff
@@ -0,0 +1,42 @@
+package gov.cabinetoffice.api.security;
+
+import com.auth0.jwt.exceptions.MissingClaimException;
+import com.auth0.jwt.exceptions.SignatureVerificationException;
+import com.auth0.jwt.exceptions.TokenExpiredException;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.NonNull;
+import org.springframework.web.filter.OncePerRequestFilter;
+
+import java.io.IOException;
+
+public class ExceptionHandlerFilter extends OncePerRequestFilter {
+    @Override
+    protected void doFilterInternal(@NonNull HttpServletRequest request, @NonNull HttpServletResponse response, @NonNull FilterChain filterChain) throws ServletException, IOException {
+        try {
+            filterChain.doFilter(request, response);
+        }
+        catch (TokenExpiredException e) {
```

Comment:
I think you could do this in one catch block then potentially use a switch statement on the exception type to set the message since thats the only thing that changes about all this duplicated code. 

```
catch (ExceptionType1 | Exceptiontype2 | ExceptionType3 ex){   
    // catch block
}